views/login/questions/trillium.pug

```diff
@@ -4,6 +4,8 @@ block variables
   -var title = __('Ontario Trillium Benefit amount and payment type')
   -var trilliumAmount = hasData(body, 'trilliumAmount') ? body.trilliumAmount : ''
   -var trilliumPaymentMethod = hasData(body, 'trilliumPaymentMethod') ? body.trilliumPaymentMethod : ''
+  -var placeholder= currencyFilter(0, getLocale()).replace('$', '')
+  -var formattedValue= (trilliumAmount !== '') ? currencyFilter(trilliumAmount, getLocale()).replace('$', '') : ''
```

Comment:
Is there an easy way to make `currencyFilter({amount}, getLocale()).replace('$', '')` into a mixin or function or something? Sucks to have to write it this way so many times.

Reply:
I can look into it